pyproject.toml

```diff
@@ -1,16 +1,79 @@
+[build-system]
+requires = [ "poetry>=1.0.3",]
+build-backend = "poetry.masonry.api"
+
+[tool.poetry]
+name = "rasa-sdk"
+version = "1.7.0"
+description = "Open source machine learning framework to automate text- and voice-based conversations: NLU, dialogue management, connect to Slack, Facebook, and more - Create chatbots and voice assistants"
+authors = [ "Rasa Technologies GmbH <hi@rasa.com>",]
+maintainers = [ "Tom Bocklisch <tom@rasa.com>",]
+homepage = "https://rasa.com"
+repository = "https://github.com/rasahq/rasa-sdk"
+documentation = "https://rasa.com/docs"
+classifiers = [ "Development Status :: 4 - Beta", "Intended Audience :: Developers", "License :: OSI Approved :: Apache Software License", "Programming Language :: Python", "Programming Language :: Python :: 3.6", "Programming Language :: Python :: 3.7", "Topic :: Software Development :: Libraries",]
```

Comment:
unfortunately, we cannot :) or, to be more specific, it makes very little sense because the file will be reformatted anyway when you change a version using `make release`